RecoHI/HiJetAlgos/python/HiRecoPFJets_cff.py

```diff
@@ -59,20 +86,52 @@
 kt4PFJetsForRho.GhostArea     = cms.double(0.005)
 
 from RecoHI.HiJetAlgos.hiFJRhoProducer import hiFJRhoProducer
+#from RecoHI.HiJetAlgos.hiPuRhoProducer_cfi import hiPuRhoProducer
+#from RecoHI.HiJetAlgos.hiFJRhoFlowModulationProducer_cfi import hiFJRhoFlowModulationProducer
+#from RecoHI.HiJetAlgos.hiPuRhoProducer import hiPuRhoProducer
+#from RecoHI.HiJetAlgos.hiFJRhoFlowModulationProducer import hiFJRhoFlowModulationProducer
+#from RecoHI.HiJetAlgos.hiPFCandCleaner_cfi import hiPFCandCleaner
+
+hiFJRhoFlowModulationProducer = cms.EDProducer(
+    'HiFJRhoFlowModulationProducer',
+    pfCandSource = cms.InputTag('particleFlow'),
+    doJettyExclusion = cms.bool(False),
+    doFreePlaneFit = cms.bool(False),
+    doEvtPlane = cms.bool(False),
+    doFlatTest = cms.bool(False),
+    jetTag = cms.InputTag("ak4PFJets"),
+    EvtPlane = cms.InputTag("hiEvtPlane"),
+    evtPlaneLevel = cms.int32(0)
+    )	
+
+hiPuRhoProducer = cms.EDProducer(
+    'HiPuRhoProducer',
+    dropZeroTowers = cms.bool(True),
+    medianWindowWidth = cms.int32(2),
+    minimumTowersFraction = cms.double(0.7),
+    nSigmaPU = cms.double(1.0),
+    puPtMin = cms.double(15.0),
+    rParam = cms.double(.3),
+    radiusPU = cms.double(.5),
+    src = cms.InputTag('PFTowers'),
+    towSigmaCut = cms.double(5.), 
+    )
```

Comment:
replace with what's generated by the fillDescriptions
 actually, I'd suggest to drop the "Producer" from the product name, which is a more common for persisted data; this would mean also a few edits in the InputTags
```suggestion
import RecoHI.HiJetAlgos.hiPuRhoProducer_cfi as _mod
hiPuRho = _mod.hiPuRhoProducer.clone()
```